src/makeRobot.js

```diff
@@ -38,7 +38,78 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      'x': 0, 'y': 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+
+      return this.coords;
+    },
+
+    goForward(step = 1) {
+      const myStep = step;
+
+      if (myStep <= 0) {
+        return this;
+      }
+
+      this.coords.y += step;
+
+      return this;
```

Comment:
```suggestion
      if (myStep > 0) {
        this.coords.y += step;
      }

      return this;
```

it will be more readable